Source/Foundation/bsfUtility/Error/BsCrashHandler.h

```diff
@@ -105,6 +131,22 @@ namespace bs
 		/** Error message to display on program failure. */
 		static const String sFatalErrorMsg;
 
+		// Handling customization callbacks
+		/** @copydoc CrashHandlerSettings::onBeforeReportCrash */
+		std::function<bool(const String& type,
+			const String& description,
+			const String& function,
+			const String& file,
+			UINT32 line)> onBeforeReportCrash;
```

Comment:
Members should have 'm' prefix (`mOnBeforeReportCrash`) and be listed before static memebers (i.e. before `sFatalErrorMsg`). Same goes for the two below.

Reply:
Actually, can't we just keep a copy of CrashHandlerSettings here? Saves copying the fields and the doc comments as well.